transfluent.py

```diff
@@ -117,7 +117,7 @@ def texts_save(self, group_id, language, texts,
             'language': language,
             'invalidate_translations': 1 if invalidate_translations else 0,
         }
-        for key, content in texts.iteritems():
+        for key, content in texts.items():
```

Comment:
This will change the behaviour on Python 2.

Create a `_compat` module that will contain things we need for Python 2/3 compatibility. Add `iteritems` function for both Python 2 and 3 there and use it here: `for key content in iteritems(texts)`. See [Flask's `_compat`](https://github.com/mitsuhiko/flask/blob/master/flask/_compat.py) module for an example.